package.json

```diff
@@ -56,5 +56,6 @@
 		"postcss-nesting": "latest",
 		"rollup": "^2.10",
 		"rollup-plugin-terser": "^7"
-	}
+	},
+	"sideEffects": ["src/index.js", "./src/spaces/index.js"]
```

Comment:
Nitpick: Can we format it in multiple lines?

Reply:
Good idea. Done!